### PR TITLE
Ui local date time format

### PR DESCRIPTION
**Pull request by potatowagon**
since master cant compile, the result here is taken off by rebasing the commits onto a recent working branch and running. Afterthat these commits are rebased back ontop of master.

screenshot 59

### CODE REVIEW

**Comment by pyokagan** [comment]
Well, master will really not compile if this PR gets merged :-)

```
seedu.address.ui.DeadlineTaskListCardTest > withDeadlineTask_showsInfoInLabels FAILED

    org.junit.ComparisonFailure: expected:<[1970-01-01T]01:00> but was:<[01/01/1970 Time: ]01:00>

        at org.junit.Assert.assertEquals(Assert.java:115)

        at org.junit.Assert.assertEquals(Assert.java:144)

        at seedu.address.ui.DeadlineTaskListCardTest.withDeadlineTask_showsInfoInLabels(DeadlineTaskListCardTest.java:50)

seedu.address.ui.EventTaskListCardTest > withEventTask_showsInfoInLabels FAILED

    org.junit.ComparisonFailure: expected:<[1970-01-01T]00:00> but was:<[01/01/1970 Time: ]00:00>

        at org.junit.Assert.assertEquals(Assert.java:115)

        at org.junit.Assert.assertEquals(Assert.java:144)

        at seedu.address.ui.EventTaskListCardTest.withEventTask_showsInfoInLabels(EventTaskListCardTest.java:53)

```

Always remember to run tests before submitting PRs.

**Comment by potatowagon** [comment]
i will wriite testnow

**Review by nightism — APPROVED** [review]
I think there is no critical issue in this PR. But I think as for the starting time and ending time of the event tasks, they can be just put in one lien, so that the card can be narrower to reduce the white space.

File: src/test/java/seedu/address/ui/DeadlineTaskListCardTest.java [hunk 1]
```diff
@@ -47,17 +46,6 @@ public void withDeadlineTask_showsInfoInLabels() throws Exception {
...
-    @Test
-    public void nullDeadlineTask_becomesInvisible() throws Exception {
-        FxToolkit.setupSceneRoot(() -> {
-            deadlineTaskListCard = new DeadlineTaskListCard(null, 34);
-            return deadlineTaskListCard.getRoot();
-        });
-        setupNodes();
-        assertFalse(deadlineTaskListCard.getRoot().isVisible());
-    }
-
```
Comment (pyokagan) [inline src/test/java/seedu/address/ui/DeadlineTaskListCardTest.java:62]: Why was this test removed?

**Comment by pyokagan** [comment]
Merged as 53bf48a1cacf8eba36a9ecbbd2df248b7bec44cf

**Pull request closed without merging**